File: backend/config/hospitalConfig.js
const text = (value) => {
  if (value === null || value === undefined) return "";
  const next = String(value).trim();
  return next.length > 0 ? next : "";
};

const hospitalName = text(process.env.FACILITY_NAME) || "Sarita Health Care";
const hospitalShortName =
  text(process.env.HOSPITAL_SHORT_NAME) || "Sarita";
const hospitalLegalName =
  text(process.env.BENEFIT_NAME) || text(process.env.HOSPITAL_LEGAL_NAME) || "SaritaInfotech";
const hipId = text(process.env.HIP_ID) || "IN2410002480";
const hiuId = text(process.env.HIU_ID) || hipId;
const requesterName =
  text(process.env.REQUESTER_NAME) || `${hospitalShortName} ABDM App`;
const scanShareHospitalName =
  text(process.env.SCAN_SHARE_HOSPITAL_NAME) || "Yashfeen Group of Hospitals";
const scanShareHipId =
  text(process.env.SCAN_SHARE_HIP_ID) || "IN2410002480";
const scanShareCounterLabel =
  text(process.env.SCAN_SHARE_COUNTER_LABEL) || "Counter: 1";

module.exports = {
  hospitalName,
  hospitalShortName,
  hospitalLegalName,
  requesterName,
  hipId,
  hiuId,
  scanShareHospitalName,
  scanShareHipId,
  scanShareCounterLabel,
};

File: lib/config/hospital_config.dart
import 'dart:convert';

class HospitalConfig {
  static const String hospitalName = 'Sarita Health Care';
  static const String hospitalShortName = 'Sarita';
  static const String hospitalLegalName = 'SaritaInfotech';
  static const String requesterName = 'Sarita ABDM App';
  static const String hipId = 'IN2410002480';
  static const String hiuId = hipId;
  static const String scanShareHospitalName = 'Yashfeen Group of Hospitals';
  static const String scanShareHipId = 'IN2410002480';
  static const String scanShareCounterLabel = 'Counter: 1';
  static const String scanShareQrAssetPath =
      'assets/scan_share/yashfeen_scan_share_qr_only.png';

  static String get appTitle => '$hospitalName ABDM';
  static String get workspaceName => '$hospitalShortName workspace';

  static Map<String, dynamic> get facilityQrPayload => {
    'hipId': hipId,
    'providerId': hipId,
    'facilityId': hipId,
    'facilityName': hospitalName,
  };

  static String get facilityQrRaw => jsonEncode(facilityQrPayload);

  static Uri get facilityQrImageUrl => Uri.https(
    'api.qrserver.com',
    '/v1/create-qr-code/',
    {'size': '260x260', 'data': facilityQrRaw},
  );
}
